source/scripting_v3/GTA/World.cs

```diff
@@ -2845,6 +2899,20 @@ public static Vector3 GetNextPositionOnSidewalk(Vector3 position)
 			return Vector3.Zero;
 		}
 
+		/// <summary>
+		/// Gets the nearest roadside point to the given coordinates.
+		/// </summary>
+		/// <param name="position">The position to check around.</param>
+		public static Vector3 GetPositionOnRoadside(Vector3 position)
```

Comment:
The signature should be `bool GetPositionOnRoadside(Vector3, [some direction enum], out Vector3)`, or 3 methods with the sig `bool GetPositionOnRoadside(Vector3, out Vector3)` (different methods for different direction specifications). Should return a bool so we can get if the method suceeeds. We should avoid happening like [what happened to `LastWeaponImpactPosition`](https://github.com/scripthookvdotnet/scripthookvdotnet/blob/ab38b8a641c15bb6b2e6dedadc912f7f68d438ac/source/scripting_v3/GTA/Entities/Peds/Ped.cs#L2115).

The doc of `GET_POSITION_BY_SIDE_OF_ROAD`:
```
//INFO: If the input node position is unsuitable, it will search 30m for a valid node.
//PARAM NOTES:	vInputNodePosition	- should be the position of a road node, 
//				iDirection			- should be a direction (0=forwards, 1=back, -1=doesn't matter)
//				vOutPositionByRoad	- the roadside position is passed back in this vector
//RETURNS: A boolean to indicate whether a position was found or not.

NATIVE FUNC BOOL GET_POSITION_BY_SIDE_OF_ROAD( VECTOR vInputNodePosition, INT iDirection, VECTOR & vOutPositionByRoad ) = "0xe245e7da125485ec"
```